neon_core/configuration/neon.yaml

```diff
@@ -18,7 +18,10 @@ stt:
   ovos-stt-plugin-vosk:
     model: /home/neon/.local/share/neon/vosk-model-small-en-us-0.15
   ovos-stt-plugin-server:
-    url: https://whisper.neonaiservices.com/stt
+    urls:
+    - https://whisper.neonaiservices.com/stt
+    - https://whisper.tigregotico.pt/stt
+    - https://stt.smartgic.io/fasterwhisper/stt
```

Comment:
@goldyfruit I pulled your server as well from the default plugin config; do you mind my including it here as a fallback?